api/src/main/java/org/ehcache/spi/persistence/package-info.java

```diff
@@ -15,7 +15,6 @@
  */
 
 /**
- * {@link org.ehcache.spi.persistence.PersistableResourceService PersistableResourceService} and other persistence
- * related SPI.
+ * {@link org.ehcache.spi.persistence.StateRepository StateRepository} and other persistence related SPI.
```

Comment:
Why did we change this?  Isn't PersistableResourceService still the "top level" class in this system?